src/pyflakes/fixes.rs

```diff
@@ -73,3 +76,97 @@ pub fn remove_unused_imports(
         ))
     }
 }
+
+/// Generate a fix to remove unused keys from format dict
+pub fn remove_unused_format_arguments_from_dict(
+    locator: &SourceCodeLocator,
+    unused_arguments: &[&str],
+    stmt: &Expr,
+) -> Result<Fix> {
+    let module_text = locator.slice_source_code_range(&Range::from_located(stmt));
+    let mut tree = match_module(&module_text)?;
+    let mut body = match_expr(&mut tree)?;
+
+    let new_dict = {
+        let Expression::Dict(dict) = &body.value else {
+            bail!("Expected Expression::Dict")
+        };
+
+        Dict {
+            lbrace: dict.lbrace.clone(),
+            lpar: dict.lpar.clone(),
+            rbrace: dict.rbrace.clone(),
+            rpar: dict.rpar.clone(),
+            elements: dict
+                .elements
+                .iter()
+                .filter_map(|e| match e {
+                    DictElement::Simple {
+                        key: Expression::SimpleString(name),
+                        ..
+                    } if unused_arguments
+                        .contains(&name.value.trim_matches(|c| c == '"' || c == '\'')) =>
```

Comment:
What case is this handling?

Reply:
This one is a bit weird, perhaps there is a better way. LibCST represents dictionary keys as SimpleStrings, so a key like `foo` is `"foo"`.

Reply:
You might be able to use `TRIPLE_QUOTE_PREFIXES` or `SINGLE_QUOTE_PREFIXES` from `src/docstrings/constants.rs`...?

Reply:
I've added a `strip_quotes_and_prefixes` instead, because `SINGLE_QUOTE_PREFIXES` doesn't handle byte strings.